src/ajv.ts

```diff
@@ -1,10 +1,18 @@
 import Ajv from 'ajv';
 import ajvKeywords from 'ajv-keywords';
 import addFormats from 'ajv-formats';
+import * as tags from 'language-tags';
 
 export const ajv = new Ajv({
 	coerceTypes: true,
 });
 
+ajv.addKeyword({
+	keyword: 'isValidLanguage',
```

Comment:
May be worth making this keyword even more specific `isValidLanguageTag` (since "English" is a valid language, but would not be a valid language tag).  You could also do `isValidIetfLanguageTag` but `I` and `l` look the same in this font so it looks like "Is Validletf Language Tag" and that makes very unhappy.